08-transport.Rmd

```diff
@@ -594,7 +660,7 @@ One growing area of interest surrounds the simulation of individual people and v
 R has the capability to model people in zones, and interface with ABM software such as NetLogo.
 Combined with its geographical capabilities, demonstrated in this book, R would seem an appropriate language for such developments to take place.
 Of course this should be done in ways that build-on and extend existing work in the area.
-R's 'ecological niche' in transport modelling software could be around the integration of detailed geographic analysis techniques with more widely known and used statistical techniques used in transport research.
+R's 'ecological niche' in transport modelling software could be around the integration of detailed geographic analysis techniques with more widely known and used statistical techniques used in transport research.<!-- and advanced statistical technical so far less commonly applied.-->
```

Comment:
wouldn't be the statistical techniques so far less commonly used in transportation a further goal?

Reply:
Addressed - the new wording advocates this.